jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -50,6 +50,24 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{conf-accId}/${role_id}/g' ./swagger/swagger.json"
 		}
 
+		if ( (service_name.trim() == "jazz_codeq") ) {
```

Comment:
@SiniWilson can you help me understand why we need to do this change

Reply:
sure.